src/pcapi/core/bookings/conf.py

```diff
@@ -94,5 +94,10 @@ class LimitConfigurationV2(BaseLimitConfiguration):
 }
 
 
-def get_current_deposit_version():
-    return 2 if feature_queries.is_active(FeatureToggle.APPLY_BOOKING_LIMITS_V2) else 1
+def get_current_deposit_version(is_apply_booking_limits_v2_active=None):
+    is_apply_booking_limits_v2_active = (
+        is_apply_booking_limits_v2_active
+        if is_apply_booking_limits_v2_active != None
+        else feature_queries.is_active(FeatureToggle.APPLY_BOOKING_LIMITS_V2)
+    )
```

Comment:
(style / détail) je trouverais un peu plus lisible la forme suivante :

```suggestion
    if is_apply_booking_limits_v2_active is None:
        is_apply_booking_limits_v2_active = feature_queries.is_active(FeatureToggle.APPLY_BOOKING_LIMITS_V2)
```

(Je trouve qu'on perd un peu en lisibilité pour gagner une requête SQL, je me demande si ça vaut le coût. Mais pourquoi pas. Et puis ce _feature flag_ va être supprimé très bientôt, de toute façon.)